### PR TITLE
Issue #1982 fixed minikube iso-url validation

### PR DESCRIPTION
**Pull request by MansM**
fixes https://github.com/minishift/minishift/issues/1982

### CODE REVIEW

**Comment by praveenkumar** [comment]
add to whitelist

**Comment by praveenkumar** [comment]
@MansM Thanks for PR, can you please update your commit message like `Issue #1982 fixed minikube iso-url validation` otherwise our commit validation CI will fail.

**Comment by MansM** [comment]
should be ok now?

**Comment by praveenkumar** [comment]
@MansM yes now looks good, let CI run and then we can have it in.

**Comment by praveenkumar** [comment]
retest this please.

File: pkg/minishift/config/validations.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func IsValidProxy(name string, uri string) error {
 }
 
 func IsValidUrl(_ string, isoURL string) error {
-	if isoURL == minishiftConstants.B2dIsoAlias || isoURL == minishiftConstants.CentOsIsoAlias {
+	if isoURL == minishiftConstants.B2dIsoAlias || isoURL == minishiftConstants.CentOsIsoAlias || isoURL == minishiftConstants.MinikubeIsoAlias {
```
Comment (praveenkumar) [inline pkg/minishift/config/validations.go]: 👍

File: pkg/minishift/config/validations_test.go [hunk 2]
```diff
@@ -134,6 +134,10 @@ func TestValidURL(t *testing.T) {
 			value:     "b2d",
 			shouldErr: false,
 		},
+		{
+			value:     "minikube",
+			shouldErr: false,
+		},
```
Comment (praveenkumar) [inline pkg/minishift/config/validations_test.go]: 👍

**Pull request merged**